app/src/main/java/org/kiwix/kiwixmobile/zim_manager/fileselect_view/ZimFileSelectFragment.java

```diff
@@ -204,8 +204,13 @@ public boolean onActionItemClicked(ActionMode mode, MenuItem item) {
                */
 
               File shareFile = new File(shareFilePath);
-              Uri shareContentUri = FileProvider.getUriForFile(getContext(),
+              Uri shareContentUri;
+              if(Build.VERSION.SDK_INT >=24)
+                shareContentUri= FileProvider.getUriForFile(getContext(),
```

Comment:
Add braces to if else statements, it will make addition of more statements easy in future.

Reply:
OK.

Reply:
@abdulwd I have to modify this commit or make a new commit

Reply:
@abdulwd I made a new commit. Is it okay?